students/jemielity_kamil/lesson_09_reading_and_writing_files/mad_libs.py

```diff
@@ -0,0 +1,18 @@
+mad_libs_file = open('mad_libs')
```

Comment:
It's a good practice to use 'with' statement for opening files in case some exception occurs before file handle is closed

```
with open('mad_libs') as mad_libs_file:
    mad_libs_content = mad_libs_file.read()
    # mad_libs_file.close() <- this will be implicitly called by __exit__() when going out of with statement
```